dolphinscheduler-dao/src/test/java/org/apache/dolphinscheduler/dao/utils/DagHelperTest.java

```diff
@@ -280,9 +280,15 @@ public void testConditionPostNode() throws IOException {
     }
 
     /**
+     * process:
      * 1->2->3->5->7
      * 4->3->6
-     * 2->8->5->7
+     * 1->2->8->5->7
+     * DAG graph:
+     *    8->5->7
+     *    ↑  ↑
+     * 1->2->3->6
+     *    4-↗
```

Comment:
I have an experience in Python api about draw a graph about DAG, and maybe we should keep the longest flow into online `1-2-3-5-7` and the finial is like

```suggestion
     *      4 ->   -> 6
     *          \ /
     * 1 -> 2 -> 3 -> 5 -> 7
     *       \       /
     *        -> 8 ->
```